release-notes/search-processor.release-notes-2.5.0.md

```diff
@@ -0,0 +1,38 @@
+Compatible with OpenSearch (2.5.0).
+
+### Features
+
+* Add a Docker quickstart script for demo / test ([#25](https://github.com/opensearch-project/search-processor/pull/25))
+* Configure Mend for GitHub.com ([#37](https://github.com/opensearch-project/search-processor/pull/37))
+
+### Enhancements
+
+* Fix codecov after repo renaming ([#41](https://github.com/opensearch-project/search-processor/pull/41))
+* stripped down version of pr stats for only this repository ([#56](https://github.com/opensearch-project/search-processor/pull/56))
+* Update log statements in quickstart helper script ([#79](https://github.com/opensearch-project/search-processor/pull/79))
+
+### Bug Fixes
+
+* Fix codecov after repo renaming ([#41](https://github.com/opensearch-project/search-processor/pull/41))
+* Bug fixes and refactoring ([#38](https://github.com/opensearch-project/search-processor/pull/38))
+* Fix permissions for PR Stats action ([#67](https://github.com/opensearch-project/search-processor/pull/67))
```

Comment:
Should this be Maintenance?

Reply:
similarly, these github actions are CIs, does it make sense falling into `Infrastructure`: A change to infrastructure, testing, CI/CD, pipelines?